src/webresource-handler/index.ts

```diff
@@ -182,7 +190,15 @@ export const getUploaderMiddlware = (
 		// multipart requests will have two main parts, the file contents and the form fields
 		// This receives the form fields and transforms them into a standard JSON body
 		// This is a similar behavior as previous multer library did
-		bb.on('field', (name, val) => {
+		bb.on('field', async (name, val) => {
```

Comment:
```suggestion
		bb.on('field', (name, val) => {
```
Nit: I don't think the `async` notation is necessary here